resources/views/email_services/options/postmark.blade.php

```diff
@@ -1 +1,2 @@
-<x-sendportal.text-field name="settings[key]" :label="__('API Key')" :value="Arr::get($settings ?? [], 'key')" autocomplete="off" />
\ No newline at end of file
+<x-sendportal.text-field name="settings[key]" :label="__('API Key')" :value="Arr::get($settings ?? [], 'key')" autocomplete="off" />
+<x-sendportal.text-field name="settings[message_stream]" :label="__('Message Stream')" :value="Arr::get($settings ?? [], 'message_stream')" autocomplete="off" />
```

Comment:
Is it worth linking to the Postmark docs or something for this, or explaining what it is? I could imagine some people being confused what they are being asked for here.

Reply:
Yes that was my original intent, but I was not satisfied by how the link looked like. So I thought to simply add the link to our documentation.